proto/src/ground/v1beta1/audit_info.proto

```diff
@@ -40,4 +39,7 @@ message AuditInfo {
 
   // URL of the user's profile picture.
   string photo_url = 5;
+
+  // The user's email address.
+  string email_address = 6;
```

Comment:
I should note that we will need to fix the app client to address this, since we were storing and querying by `user_id`, which may constitute a large/breaking change...

Can you file a bug on the Android side and I can quickly get this fixed?

Reply:
Querying for what, exactly? Ad hoc LOIs? If so, wouldn't that continue to work, since this is just an additive change?

Reply:
Ooh... I must have gotten the Survey's ACL field confused with AuditInfo. I think we should be ok! We'll just want to make sure we're clear which field is meant to be the primary key etc.